CMakeLists.txt

```diff
@@ -184,6 +184,9 @@ INSTALL(TARGETS smack
 INSTALL(FILES ${CMAKE_CURRENT_SOURCE_DIR}/bin/boogie
   ${CMAKE_CURRENT_SOURCE_DIR}/bin/corral
   ${CMAKE_CURRENT_SOURCE_DIR}/bin/smack.py
+  ${CMAKE_CURRENT_SOURCE_DIR}/bin/tokenizer
+  ${CMAKE_CURRENT_SOURCE_DIR}/bin/token_replace.py  
```

Comment:
Please see #99  for a proposal on how to handle this clutter in the `bin/` installation directory.